request_a_govuk_domain/request/forms.py

```diff
@@ -123,9 +118,7 @@ def clean_file(self):
         2. Content Type
         """
         file = self.cleaned_data.get('file')
-        file_content = file.content_type.split('/')[0]
-
-        if file_content in settings.CONTENT_TYPES:
+        if file is not None and file.content_type.split('/')[0] in settings.CONTENT_TYPES:
```

Comment:
I think the file would have some value by this time, otherwise front-end will throw an error , that would be because of this code:
```
error_messages={
            "required": "Choose the file you want to upload."
        },
```
hence `not None` check is not needed

Reply:
I added this because static type checking picked it as string or None, and so is asking if we can check if none. It's a choice people often need to make generally when using static type checking, and my opinion is to check everything as None values might come from other functions we don't know about. What's the problem with having a check for None?